_posts/2023-10-10-endless-os.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: distro
+author: "Sam Hewitt"
+updater: "cassidyjames"
+title: "Endless OS"
+date: "2013-09-14"
+updated: "2023-10-10"
+logo: "endlessos"
+image: "endlessos-5.png"
```

Comment:
Shouldn't we delete the previous `endlessos-3.4.png` file from the repo, since it's no longer used by any page (AFAICT)?

Reply:
Can do, but it looked like old assets were still around for other releases? I'm indifferent, though.

Reply:
Oh, nice catch! I hadn't noticed that. I suppose it's nice to keep the old screenshots around so we can compare a distro's evolution over time. Let's keep them, then.